tests/test_cli.py

```
@@ -0,0 +1,952 @@
+        mock_print.assert_called()
+
+    async def _stub_interactive_login_success(*_a, **_k):
+        return True
+
+    @patch("os.path.exists", return_value=True)
+    @patch("sys.argv", ["biblebot", "auth", "login"])
+    @patch("builtins.input", return_value="https://matrix.org")
+    @patch("getpass.getpass", return_value="password")
+    @patch("biblebot.auth.interactive_login", new=_stub_interactive_login_success)
+    @patch("sys.exit")
+    def test_auth_login_command(self, mock_exit, mock_getpass, mock_input, mock_exists):
```

Ruff (0.12.2)

Unused method argument: `mock_getpass`

(ARG002)